### DISCUSSION TITLE
nyc platform

### DISCUSSION THREAD

**Discussion by yashu1wwww** [community/community#51728] [Other Feature Feedback, Questions, & Ideas]
code

**Comment by queenofcorgis** [comment]
Hi @yashu1wwww, we're happy you're here! You are more likely to get a useful response if you are explicit about what your project entails, giving a few more details might help someone give you a nudge in the right direction, and post your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by yashu1wwww** [comment]
wt issue
